docs/accessories/developer/itemstack_components.md

```diff
@@ -0,0 +1,284 @@
+---
+title: ItemStack Data Components
+project: accessories
+---
+
+## General
+
+Below is the current set of available Data Components for ItemStacks that are supported with documentation about there function and format.
+
+### Attribute Modifiers
+
+Any ItemStack can have Attributes added onto such though the `accessories:attributes` item component with such being comprised of a list of attribute entries similar to Minecraft's default implementation. Below is an example of such:
+
+```JSON
+{
+    "accessories:attributes": {
+        "modifiers": [
+            {
+                "type": "minecraft:attack_damage",
+                "id": "example:increased_damage_boost",
+                "amount": 1,
+                "operation": "add_value",
+                "slot_name": "hat",
+                "is_stackable": false,
+            }
+        ]
+    }
+}
+```
+
+| <div style="width:118px">Field</div> | Usage |
+|--|--|
+|`"type"`| Refers to the attribute type to which should be adjusted with the given modifier data either registered or a slot attribute. |
+|`"id"`| The unique resource location to identify this modifier |
+|`"amount"`| The Amount of change from the modifier |
+|`"operation"`| The type of calculation operation that should be performed out of the valid values. |
+|`"slot_name"`| The slot to which must be equipped for the effect to apply or `"any"` if it's based on where it can and can't be equipped |
+|`"is_stackable"`| Indicates that if the given `"id"` passed should be appended with slot information to allow for additional copies of the attribute to function together as a compounding effect |
+
+### Slot Validation
+
+Any ItemStack can define invalid or valid slots to which it can be equipped within the `accessories:slot_validation` item component allowing you to add the ability to equip such within a slot or override current equability for an existing accessory.
```

Comment:
"Any ItemStack can define invalid or valid slots to which it can be equipped using the `accessories:slot_validation` item component, allowing you to add the ability to equip the accessory within a slot or override current equipability for an existing accessory."